doc/configure-device.md

```diff
@@ -57,53 +66,37 @@ You need a Linux PC (for flashing), a Raspberry Pi 4 board, and a micro SD card
    └─sdc1        8:33   1  29.3G  0 part
    ```
 
-1. Download a flashing script (`sd_fusing_rpi3.sh`) from one of the following URLs.
-
-   - https://git.tizen.org/cgit/platform/kernel/u-boot/plain/scripts/tizen/sd_fusing_rpi3.sh?h=tizen
-   - https://review.tizen.org/git/?p=platform/kernel/u-boot.git;a=blob_plain;f=scripts/tizen/sd_fusing_rpi3.sh;hb=refs/heads/tizen
-
-   Note: You may notice that the repository also contains a script for Pi 4 (`sd_fusing_rpi4.sh`). Do not use it.
-
-1. Re-partition the SD card (replace `/dev/sdc` with your own device name):
+1. Download a fusing script(`sd_fusing.py`).
 
    ```sh
-   $ chmod 755 sd_fusing_rpi3.sh
-   $ sudo ./sd_fusing_rpi3.sh -d /dev/sdc --format
-
-   Raspberry Pi downloader, version 2.1.0
 
-   Device: /dev/sdc
-
-   /dev/sdc will be formatted, Is it OK? [y/<n>]
+   $ wget "https://review.tizen.org/git/?p=platform/kernel/tizen-fusing-scripts.git;a=blob_plain;f=scripts/sd_fusing.py;hb=HEAD" --output-document=sd_fusing.py
+   $ chmod +x sd_fusing.py
+   ```
 
-   ...
+1. Re-partition the SD card. (replace `/dev/sdc` with your own device name):
 
-   End /dev/sdc format
+   ```sh
+   $ sudo ./sd_fusing.py -d <SD card device name> -t <Target Board> --format
+   ```
+   For example:
+   ```sh
+   $ sudo ./sd_fusing.py -d /dev/sdc -t rpi4 --format
    ```
 
-1. Download a Tizen OS image. Visit the links below and download two `.tar.gz` files.
+1. [Download a Tizen OS image](https://docs.tizen.org/platform/developing/flashing-rpi/#download-binaries). Visit the links below and download two `.tar.gz` files.
 
-   - [tizen-boot-arm64-rpi4](http://download.tizen.org/releases/milestone/TIZEN/Tizen/Tizen-Unified/latest/images/standard/tizen-boot-arm64-rpi4)
-   - [tizen-headed-aarch64](http://download.tizen.org/releases/milestone/TIZEN/Tizen/Tizen-Unified/latest/images/standard/tizen-headed-aarch64)
+   - [tizen-boot-armv7l-rpi4](https://download.tizen.org/releases/milestone/TIZEN/Tizen/Tizen-Unified/latest/images/standard/tizen-boot-armv7l-rpi4/)
```

Comment:
Do we default to arm32? I thought the arm64 image would perform better on the arm64 hardware.

Reply:
Matched the official guide document. https://docs.tizen.org/platform/developing/flashing-rpi/